### PR TITLE
rgxg: Init at 0.1.2

### PR DESCRIPTION
**Pull request by hloeffler**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/text/rgxg/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  };
+
+  outputs = [ "bin" "doc" "man" "dev" "lib" "out" ];
+
+ doInstallCheck = true;
+ installCheckTarget = "check";
+
+ postInstallCheck = ''
+   $bin/bin/rgxg help >/dev/null
+ '';
```
Comment (doronbehar) [inline pkgs/tools/text/rgxg/default.nix:19]: Fix indentation

File: pkgs/tools/text/rgxg/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  src = fetchurl {
+    url = "https://github.com/rgxg/rgxg/releases/download/v${version}/rgxg-${version}.tar.gz";
+    sha256 = "554741f95dcc320459875c248e2cc347b99f809d9555c957d763d3d844e917c6";
+  };
+
+  outputs = [ "bin" "doc" "man" "dev" "lib" "out" ];
+
+ doInstallCheck = true;
+ installCheckTarget = "check";
```
Comment (doronbehar) [inline pkgs/tools/text/rgxg/default.nix:15]: What's wrong with regular `doCheck` and `checkTarget`?

File: pkgs/tools/text/rgxg/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ '';
+
+  meta = with stdenv.lib; {
+    description = ''rgxg (ReGular eXpression Generator) is a C library and a command-line tool
+    to generate (extended) regular expressions.'';
+    license = licenses.zlib;
+    maintainers = with maintainers; [ hloeffler ];
+    homepage = "https://rgxg.github.io/";
+    downloadPage = "https://github.com/rgxg/rgxg/releases";
+    inherit version;
```
Comment (doronbehar) [inline pkgs/tools/text/rgxg/default.nix:28]: Not needed.

```suggestion
```

File: pkgs/tools/text/rgxg/default.nix [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ doInstallCheck = true;
+ installCheckTarget = "check";
+
+ postInstallCheck = ''
+   $bin/bin/rgxg help >/dev/null
+ '';
+
+  meta = with stdenv.lib; {
+    description = ''rgxg (ReGular eXpression Generator) is a C library and a command-line tool
+    to generate (extended) regular expressions.'';
```
Comment (doronbehar) [inline pkgs/tools/text/rgxg/default.nix:23]:
```suggestion
    description = "A C library and a command-line tool to generate (extended) regular expressions";
```

**Review by doronbehar — CHANGES_REQUESTED** [review]
Also, your PR should have eventually two commits:

```
maintainers: add <me>
<package-name>: init at <version>
```

File: pkgs/tools/text/rgxg/default.nix (hunk 2 above)
Comment (hloeffler) [reply pkgs/tools/text/rgxg/default.nix:15]: nothing,
i don't understand everything yet, this was copied from an other package, and i was just happy that it worked

**Comment by hloeffler** [comment]
Thank you for your time and feedback,
i will find out how to do it

File: pkgs/tools/text/rgxg/default.nix (hunk 2 above)
Comment (doronbehar) [reply pkgs/tools/text/rgxg/default.nix:15]: I see you are new :) Welcome aboard. I don't know where you borrowed that from, but it's considered more common to use `doCheck = true`. `check` is the default check (make) target for `doCheck`, so you should spare yourself 1 line of from the expression. For more info, refer to https://nixos.org/nixpkgs/manual/#chap-stdenv .

Comment (hloeffler) [reply pkgs/tools/text/rgxg/default.nix:15]: I don't know how to edit my commits here, so i created a new pr with 2 commits:
https://github.com/NixOS/nixpkgs/pull/99422

Thanks for the weclome

**Comment by hloeffler** [comment]
i created a new one: https://github.com/NixOS/nixpkgs/pull/99422

**Pull request closed without merging**